recipes/opencv/4.x/conanfile.py

```diff
@@ -1101,21 +1101,21 @@ def requirements(self):
         if self.options.get_safe("with_gtk"):
             self.requires("gtk/system")
         if self.options.get_safe("with_qt"):
-            self.requires("qt/5.15.11")
+            self.requires("qt/6.6.1")
```

Comment:
You're more knowledgeable than me in qt related things, so I'm taking that v6 is ok for opencv? Did you get a chance to locally test it?

Reply:
No, my tests did not go further than the test recipe

Reply:
mmm, this is not compatible with https://github.com/conan-io/conan-center-index/blob/3c36bb86edf99b4c1d18511263417a3682c679ba/recipes/opencv/4.x/patches/4.5.5-0004-link-qt-targets.patch at least. I'll stay on qt5 then